modules/openapi-generator/src/main/resources/dart2/pubspec.mustache

```diff
@@ -9,15 +9,15 @@ authors:
   - '{{{pubAuthor}}} <{{{pubAuthorEmail}}}>'
 homepage: '{{{pubHomepage}}}'
 environment:
-  sdk: '>=2.0.0 <3.0.0'
+  sdk: '>=2.12.0 <3.0.0'
 dependencies:
-  http: '>=0.12.0 <0.13.0'
-  intl: '^0.16.1'
+  http: '>=0.13.0 <0.14.0'
+  intl: '^0.17.0'
   meta: '^1.1.8'
 {{#json_serializable}}
-  json_annotation: '^3.1.1'{{/json_serializable}}
+  json_annotation: '^4.0.1'{{/json_serializable}}
 dev_dependencies:
-  test: '>=1.3.0 <1.16.0'
+  test: '>=1.3.0 <1.18.0'
```

Comment:
This still needs a lower bound of `>=1.16.0`

Reply:
why?

Reply:
Because https://pub.dev/packages/test/changelog#1160-nullsafety8 is the first version that fully support running on Dart 2.12 which is now required. Anything below 1.16.0 will not compile.